client/app/bundles/Attendance/components/ClassScreen/ClassScreen.jsx

```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I just recently saw that they moved PropTypes out. I think, in the future, if we're going to change a few instances, I'd rather keep things consistent and do them the old way, or replace all instances of it. For now, this is fine, I'll make sure I update the rest as I poke around more.

Reply:
I know I had the same thought. Didn't wanna do it in this branch, but we should do it eventually